### DISCUSSION TITLE
where are the official stuff?

### DISCUSSION THREAD

**Discussion by ViBE-HU** [rehlds/ReVoice#22] [Q&A]
i know there's the code but i cannot compile it so. where do you publish the official binary? and are there any official thread for information/discuss/support?

**Comment by Adidasman1** [comment]
yes, http://www.dedicated-server.ru/vbb/showthread.php?t=26598

**Comment by ViBE-HU** [comment]
@Adidasman1 are you sure? the registration method is a nightmare. why is it so necessary to being a geek...

**Comment by hajimura** [comment]
@ViBE-HU, binaries are published here too: https://neugomon.ru/threads/730/

**Comment by ViBE-HU** [comment]
thank's guys. i used @Adidasman1 's suggestion. other forum's limitations make no sense.
